File: Serene/Serene.Web/Modules/Common/Imports/ServerTypings/Administration.LanguageForm.ts
namespace Serene.Administration {
    export class LanguageForm extends Serenity.PrefixedContext {
        static formKey = 'Administration.Language';

    }

    export interface LanguageForm {
        LanguageId: Serenity.StringEditor;
        LanguageName: Serenity.StringEditor;
    }

    [['LanguageId', Serenity.StringEditor], ['LanguageName', Serenity.StringEditor]].forEach(x => Object.defineProperty(LanguageForm.prototype, <string>x[0], { get: function () { return this.w(x[0], x[1]); }, enumerable: true, configurable: true }));
}

File: Serene/Serene.Script/Imports/ServerImports/Administration.LanguageForm.cs
using jQueryApi;
using Serenity;
using Serenity.ComponentModel;
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace Serene.Administration
{
    public partial class LanguageForm : PrefixedContext
    {
        [InlineConstant] public const string FormKey = "Administration.Language";

        public LanguageForm(string idPrefix) : base(idPrefix) {}

        public StringEditor LanguageId { [InlineCode("{this}.w('LanguageId', Serenity.CodeGeneration.ExternalType)")] get; private set; }
        public StringEditor LanguageName { [InlineCode("{this}.w('LanguageName', Serenity.CodeGeneration.ExternalType)")] get; private set; }
    }
}

